java/src/main/java/com/google/cloud/dataproc/templates/databases/README.md

```diff
@@ -14,10 +14,18 @@ bin/start.sh \
 --templateProperty spanner.gcs.input.database.id=<database-id> \
 --templateProperty spanner.gcs.input.table.id=<table-id> \
 --templateProperty spanner.gcs.output.gcs.path=<gcs-path> \
---templateProperty spanner.gcs.output.gcs.saveMode=<Append|Overwrite|ErrorIfExists|Ignore>
---templateProperty spanner.gcs.output.gcs.format=<avro|csv|parquet|json|orc>
+--templateProperty spanner.gcs.output.gcs.saveMode=<Append|Overwrite|ErrorIfExists|Ignore> \
+--templateProperty spanner.gcs.output.gcs.format=<avro|csv|parquet|json|orc> \
+--templateProperty spanner.gcs.output.gcs.partitionColumn=<optional-partition-column> \
```

Comment:
we don't want to have `output.gcs.partitionColumn`

Reply:
Removed.